profiling/src/profiling/uploader.rs

```diff
@@ -97,21 +96,12 @@ impl Uploader {
 
         let serialized =
             profile.serialize_into_compressed_pprof(Some(message.end_time), message.duration)?;
-        let endpoint_counts = Some(&serialized.endpoints_stats);
-        let start = serialized.start.into();
-        let end = serialized.end.into();
-        let files = &[File {
-            name: "profile.pprof",
-            bytes: serialized.buffer.as_slice(),
-        }];
```

Comment:
I assume these things all moved into the internals?

Reply:
Exactly. Just the line above (the call to `profile.serialize_into_compressed_pprof()`) returns an `EncodedProfile` which has all those values already and the new API with libdatadog 17 takes this `EncodedProfile` as the first argument to `exporter.build()` instead.